cypress/e2e/clue/branch/student_tests/diagram_tool_spec.js

```diff
@@ -1,10 +1,16 @@
 import ClueCanvas from '../../../../support/elements/clue/cCanvas';
 import DiagramToolTile from '../../../../support/elements/clue/DiagramToolTile';
 import DrawToolTile from '../../../../support/elements/clue/DrawToolTile';
+import TextToolTile from '../../../../support/elements/clue/TextToolTile';
 
 let clueCanvas = new ClueCanvas,
   diagramTile = new DiagramToolTile,
-  drawTile = new DrawToolTile;
+  drawTile = new DrawToolTile,
+  textTile = new TextToolTile;
+
+const cmdKey = Cypress.platform === "darwin" ? "cmd" : "ctrl";
+const undoKeystroke = `{${cmdKey}}z`;
+const redoKeystroke = `{${cmdKey}}{shift}z`;
```

Comment:
I tried moving this into commands.js so it could potentially be reused elsewhere, but could not get that to work.